Sources/FluentKit/Query/Builder/QueryBuilder+Join.swift

```diff
@@ -12,6 +12,33 @@ extension QueryBuilder {
         self.join(Foreign.self, filter.foreign, to: Local.self, filter.local , method: method)
     }
 
+    func join<From, To>(
+        parent: KeyPath<From, ParentProperty<From, To>>,
+        method: DatabaseQuery.Join.Method = .inner
+    ) -> Self {
+        join(To.self, on: parent.appending(path: \.$id) == \To._$id, method: method)
+    }
+
+    func join<From, To>(
```

Comment:
Docblocks for all new public methods would be great. Especially for these new methods, I think a docblock explaining what join is being performed would be helpful. For the `siblings` case we should also mention that two joins are happening.